recipes/mne-realtime/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "mne-realtime" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 4e67afd386589884318a557e35156552ee46f1bd181865cd4389c927aba317d7
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - numpy
+    - scipy
+    - mne-base
+
+test:
+  imports:
+    - mne_realtime
+  requires:
+    - pip
+  commands:
+    - pip check
+
+about:
+  home: https://mne.tools/mne-realtime
+  license: BSD-3-Clause
```

Comment:
This file seems to be from a different repo under a different license: https://github.com/mne-tools/mne-realtime/blob/main/mne_realtime/externals/FieldTrip.py

Its license either needs to be added here or the vendored version removed and the package added as a dependency.

Reply:
@larsoner Could you take a look at this? I wasn't aware of this vendored module … @BastianZim's got eagle eyes! 🦅 👍 😮

Reply:
A fun little rabbit hole... It's also BSD 3-clause. I'll push out a 0.1.3 with this info

https://github.com/fieldtrip/fieldtrip/blob/master/realtime/src/buffer/python/FieldTrip.py
https://github.com/fieldtrip/fieldtrip/blob/master/realtime/src/buffer/COPYING
https://github.com/fieldtrip/fieldtrip/blob/master/realtime/src/buffer/LICENSE_BSD

Reply:
Not sure how much work it is but before you continue with this, from a conda-forge perspective having it as a separate dependency would be better instead of it being vendored. 
But if that's not possible we can work around that as well. :)

Reply:
> But if that's not possible we can work around that as well. :)

Agreed it would be better, but in this case it's not really our code to maintain, and I doubt the original authors are interested in maintaining a conda-forge package. So it would be a lot more (political and coding) work to separate it I think -- but if there are other packages that end up depending on it, agreed it will be worth separating then!

Reply:
Yeah, I just saw that it's not a proper package and only a script so no problem there then. Then let's continue with the workaround.

Reply:
Upstream lists multiple licenses based on the situation you're using it in. These would all need to be included:

https://github.com/fieldtrip/fieldtrip/blob/7d4608900c0754decbbfe24b1072901ea7b040de/realtime/src/buffer/COPYING#L5

Reply:
It might be possible to use just one if that's preferred but then I'll just quickly ping conda-forge/core for verification as they have the final decision for licenses.

Reply:
I tried to follow the instructions in COPYING, which were to 1) choose one license:

https://github.com/fieldtrip/fieldtrip/blob/7d4608900c0754decbbfe24b1072901ea7b040de/realtime/src/buffer/COPYING#L7

Then, 2) add it to your codebase, and 3) acknowledge in the manual, which for us comes from the docstring here:

https://github.com/mne-tools/mne-realtime/blob/569c56ab80ac320e92401685ca62d7ffb5aff949/mne_realtime/fieldtrip_client.py#L57-L59

So I think we should be in the clear on this one, but I am no license expert...

Reply:
Yes you're right, should be sufficient.